spec/factories.rb

```diff
@@ -1,8 +1,13 @@
 FactoryGirl.define do
   factory :user, class: Stormpath::Resource::Account do
-    email 'jlc@example.com'
+    email { Faker::Internet.email }
```

Comment:
@cilim can you rename this factory to account, it was named badly in the first place.